packages/addons/service/locale/package.mk

```diff
@@ -16,13 +16,11 @@ PKG_ADDON_TYPE="xbmc.service"
 
 addon() {
   mkdir -p "$ADDON_BUILD/$PKG_ADDON_ID/bin"
-  cp -PR "$(get_build_dir glibc)/.$TARGET_NAME/locale/localedef" \
+  cp -PR "$(get_install_dir glibc)/.noinstall/localedef" \
          "$ADDON_BUILD/$PKG_ADDON_ID/bin"
 
-  mkdir -p "$ADDON_BUILD/$PKG_ADDON_ID/i18n"
-  cp -PR "$(get_build_dir glibc)/localedata/charmaps" \
-         "$(get_build_dir glibc)/localedata/locales" \
```

Comment:
Don't we need charmaps and locales?

Reply:
Those are supposed to come from the `usr/share/i18n` folder copied below, but that doesn't happen... will fix